packages/formik/src/Field.tsx

```diff
@@ -73,9 +73,9 @@ export type FieldAttributes<T> = GenericFieldHTMLAttributes &
 
 export type FieldHookConfig<T> = GenericFieldHTMLAttributes & FieldConfig<T>;
 
-export function useField<Val = any>(
+export function useField<Val = any, Err extends {} = string>(
```

Comment:
In line with comment below, the default type here and elsewhere might be changed to?

```suggestion
export function useField<Val = any, Err extends {} = FormikErrors<Val> | string>(
```